forge/src/multi_runner.rs

```diff
@@ -548,6 +549,43 @@ mod tests {
         }
     }
 
+    #[test]
+    fn test_fork() {
+        let mut runner = runner();
+        
+        // set up all the forking stuff 
+        let chain_id = 1;
+        let block = 14444444;
+        let cache_path = Config::foundry_block_cache_file(chain_id,block);
+        let rpc = "https://rpc.flashbots.net";
+
+        runner.fork = Some(Fork {
+            cache_path,
+            url: rpc.to_string(),
+            pin_block: Some(block),
+            chain_id
+        });
+
+        runner.evm_opts.fork_block_number = Some(block);
+        runner.evm_opts.fork_url = Some(rpc.to_string());
+        runner.evm_opts.env.chain_id = Some(chain_id);
+
+        println!("{:#?}", runner.evm_opts);
+
+        let suite_result = runner.test(&Filter::new(".*", ".*", ".*fork"), None, true).unwrap();
+        for (_, SuiteResult { test_results, .. }) in suite_result {
+            for (test_name, result) in test_results {
+                println!("{}", test_name); 
```

Comment:
I was debugging and figured that this should print, but it seems like when I call `runner.test()` it returns an empty map.